omnibus/files/private-chef-cookbooks/private-chef/recipes/elasticsearch.rb

```diff
@@ -137,5 +137,10 @@
   to elasticsearch_conf_dir
 end
 
+directory '/opt/opscode/sv/opscode-solr4' do
```

Comment:
💭 We think we might not need this now that we have the prefligth check that makes sure only 1 is enabled.